relayer/cmd/run/beacon/command.go

```diff
@@ -102,3 +108,27 @@ func run(_ *cobra.Command, _ []string) error {
 
 	return nil
 }
+
+func startHttpServer(ctx context.Context) error {
+	srv := &http.Server{Addr: ":8080", Handler: nil}
```

Comment:
Make the port a command line flag which defaults to 8080.

Reply:
I ended up removing the http service and just checked if the beacon relayer process is running.